appeals/web/src/server/appeals/appeal-details/interested-party-comments/interested-party-comments.mapper.js

```diff
@@ -75,35 +75,132 @@ function generateTableRows(items, isReview = false) {
  * @param {Appeal} appealDetails
  * @param {Representation} comment
  * @returns {PageContent}
- * */
+ */
 export function viewInterestedPartyCommentPage(appealDetails, comment) {
 	const shortReference = appealShortReference(appealDetails.appealReference);
+	const commentSummaryList = generateCommentSummaryList(comment);
+	const withdrawLink = generateWithdrawLink();
+
+	const pageContent = {
+		title: 'View comment',
+		backLinkUrl: `/appeals-service/appeal-details/${appealDetails.appealId}/interested-party-comments`,
+		preHeading: `Appeal ${shortReference}`,
+		heading: 'View comment',
+		headingClasses: 'govuk-heading-l',
+		pageComponents: [commentSummaryList, withdrawLink]
+	};
+
+	return pageContent;
+}
+
+/**
+ * @param {Appeal} appealDetails
+ * @param {Representation} comment
+ * @returns {PageContent}
+ */
+export function reviewInterestedPartyCommentPage(appealDetails, comment) {
+	const shortReference = appealShortReference(appealDetails.appealReference);
+	const commentSummaryList = generateCommentSummaryList(comment);
+	const withdrawLink = generateWithdrawLink();
+
+	/** @type {PageComponent} */
+	const siteVisitRequestCheckbox = {
+		type: 'checkboxes',
+		parameters: {
+			name: 'site-visit-request',
+			items: [
+				{
+					text: 'The comment includes a site visit request',
+					value: 'site-visit',
+					checked: false
+				}
+			]
+		}
+	};
 
-	const attachmentsList = (() => {
-		if (comment.attachments.length === 0) {
-			return null;
+	/** @type {PageComponent} */
+	const commentValidityRadioButtons = {
+		type: 'radios',
+		parameters: {
+			name: 'status',
+			fieldset: {
+				legend: {
+					text: 'Do you accept the comment?',
+					isPageHeading: false,
+					classes: 'govuk-fieldset__legend--m'
+				}
+			},
+			items: [
+				{
+					value: 'valid',
+					text: 'Comment valid',
+					checked: comment?.status === 'valid'
```

Comment:
It would be nice if the status values could come from a constant (or maybe an endpoint) but I'm not sure that exists yet